server/routes/browser.py

```diff
@@ -27,3 +30,18 @@ def kg_main():
 @bp.route('/<path:dcid>')
 def kg_entity(dcid):
     return render_template('/browser/kg_entity.html', dcid=dcid)
+
+
+@bp.route('/browser2/<path:dcid>')
+def browser(dcid=None):
+    # TODO(chejennifer): Permit production use after development finishes
+    # and update routes (make sure to update routes in the React components as well)
+    if os.environ.get('FLASK_ENV') == 'production':
+        flask.abort(404)
+    node_name_values = dc.get_property_values([dcid], "name").get(dcid, [])
```

Comment:
Also can use https://github.com/datacommonsorg/website/blob/99bc65c31ea19c1d72b8b8a1fa667941a3dedf8a/server/routes/api/place.py#L121

Note need to move that function to a new file (shared library)

Reply:
Will extract the function in a separate PR